impl/src/main/java/org/jboss/forge/roaster/model/ast/AnnotationAccessor.java

```diff
@@ -149,6 +149,29 @@
       return target;
    }
 
+   public void removeAnnotations(final ASTNode body)
+   {
+      removeAnnotation(getModifiers(body));
+
+   }
+
+   public void removeAnnotations(final SingleVariableDeclaration variableDeclaration)
+   {
+      removeAnnotation(variableDeclaration.modifiers());
+   }
+
+   private void removeAnnotation(final List<?> modifiers)
+   {
+      List<?> modifiersCopy = new ArrayList<Object>(modifiers);
+      for (Object object : modifiersCopy)
```

Comment:
You can use modifiers.iterator() and call iterator.remove() instead of creating a new list. That would avoid ConcurrentModification exceptions